mozilla_vpn/views/subscriptions.view.lkml

```diff
@@ -180,6 +180,22 @@ view: +subscriptions {
   measure: count {
     type: count
   }
+
+  measure: annual_recurring_revenue {
+    description: "Annual Recurring Revenue"
+    type: sum_distinct
+    sql_distinct_key: ${subscription_id};;
+    sql: CASE
+    WHEN
+      ${plan_interval} = "year"
+    THEN
+      1 / ${plan_interval_count}
+    WHEN
+      ${plan_interval} = "month"
+    THEN
+      12 / ${plan_interval_count}
+    END * ${plan_amount} * (1 - IFNULL(${vat_rates.vat}, 0)) / 100;;
```

Comment:
This can't be done in a custom measure, because custom measures are not allowed to use fields from multiple views, and can't use sum_distinct.

Reply:
Thanks for the note.  This is really helpful to know.